docs/guides/placeholder-query-data.md

```diff
@@ -16,20 +16,15 @@ There are a few ways to supply placeholder data for a query to the cache before
 ### Placeholder Data as a Value
 
 ```js
-const result = useQuery("todos", () => fetch("/todos"), {
-  placeholderData: placeholderTodos,
-});
-```
-
-### Placeholder Data as a Function
```

Comment:
It doesn't make sense in Vue to provide placeholderData as a function because composables in Vue are executed only once (per component instance).

Hence, I think, it doesn't worth it mentioning such functionality